### DISCUSSION TITLE
Problems  adding HA-Package

### DISCUSSION THREAD

**Discussion by TrZiegler** [alexdelprete/HA-NeoPool-MQTT#17] [Q&A]
Have tried to install Your HA-Neopool-MQTT Package. Added to Packages. Now I see an error:Setup failed for 'packages': Integration not found. No entities found. Whats going wrong?

**Comment by alexdelprete** [comment]
Did you follow the link provided here? It's the docs that explain how HA manages packages:

![image](https://github.com/alexdelprete/HA-NeoPool-MQTT/assets/7027842/06f3a199-2905-46d4-baeb-00d78bb19b57)
